src/main/java/cafe/bean/Preorder.java

```diff
@@ -0,0 +1,110 @@
+package cafe.bean;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "Preorder")
+public class Preorder {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+    @OneToOne(mappedBy = "preorderId")
```

Comment:
If You change the variable name in the Review class to the suggested, don't forget to change in to @OneToOne(mappedBy = "preorder")